packages/falso/src/lib/core/unique-validators.ts

```diff
@@ -0,0 +1,32 @@
+export const primitiveValueIsUnique: <T>(item: T, items: T[]) => boolean = (
```

Comment:
WDYT about converting these inline to functions? I think it's clearer instead of the type signature and than the declaration.

Reply:
Yeah, I think you are right. Real functions will be much nicer to read